<?xml version="1.0"?>

<!-- Copyright (C) 2019-2021 LEIDOS.
Licensed under the Apache License, Version 2.0 (the "License"); you may not
use this file except in compliance with the License. You may obtain a copy of
the License at
http://www.apache.org/licenses/LICENSE-2.0
Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
License for the specific language governing permissions and limitations under
the License.
-->

<launch>
    <!-- New Guidance Commands Node -->
        <node pkg="guidance_command_repeater" type="guidance_command_repeater" name="guidance_command_repeater">
            <rosparam command="load" file="$(find guidance_command_repeater)/config/default.yaml" />
            <remap from="/cmd_speed" to="/controller/cmd_speed"/>
            <remap from="/cmd_longitudinal_effort" to="/controller/cmd_longitudinal_effort"/>
            <remap from="/cmd_lateral" to="/controller/cmd_lateral"/>
        </node>
</launch>